docs/articles/nunit-engine/extensions/AvailableExtensions.md

```diff
@@ -0,0 +1,91 @@
+---
+uid: availableengineextensions
+---
+
+# Available NUnit Engine Extensions
+
+The NUnit team provides several extensions for the engine. They are all available individually as
+packages for installation through nuget and also as chocolatey packages. Some extensions are also
+bundled with specific distributions of the Console runner.
+
+## V2 Result Writer
+
+The V2 Result writer makes it possible to save test results in the older NUnit V2 format, for use
+with report programs and integration servers, which require that format.
+
+### Installation
+
+* NUget: Reference package `NUnit.Extension.NUnitV2ResultWriter` in your project
+* Chocolatey: `choco install nunit-extension-v2-result-writer`
+* Also bundled with the `NUnit.Console` NuGet and msi packages.
+
+### Usage
+
+ Specify `nunit2` format with the console runner `--result` or `--explore` options.
+
+## V2 Framework Driver
+
+The V2 Framework Driver allows the engine to run NUnit V2 tests, which is not normally possible.
+
+### Installation
+
+* NuGet: Reference package `NUnit.Extension.NUnitV2.Driver` in your project
+* Chocolatey: `choco install nunit-extension-nunit-v2-driver`
+* Also bundled with the `NUnit.Console` NuGet and msi packages.
+
+### Usage
+
+ If installed, it is used automatically when V2 tests are being run.
+
+## NUnit Project Loader
+
+The NUnit Project Loader allows the engine to run NUnit projects, file extension `.nunit`.
+If it is not installed, the runner will give an error indicating the file type is unrecognized.
+
+### Installation
+
+* NuGet: Reference package `NUnit.Extension.NUnitProjectLoader` in your project
+* Chocolatey: `choco install nunit-extension-nunit-project-loader`
+* Also bundled with the `NUnit.Console` NuGet and msi packages.
+
+### Usage
+
+ Create the project in any editor, following the NUnit project format specification.
+ You may also use the NUnit Project Editor, a legacy V2 program, since the format has
+ not changed in NUnit 3.
+
+## Visual Studio Project Loader
+
+The VS Project Loader allows the engine to load tests from a Visual Studio
+project or solution. Note that when used with a solution file, the extension
+tries to avoid loading assemblies, which do not contain tests, through use
+of heuristics, which may fail in certain circumstances.
```

Comment:
We have a note format we can use here for a nicer visual, I think. I'll look it up and make the suggestion.